cms/envs/common.py

```diff
@@ -2308,6 +2308,7 @@
 #   the orgs referenced in Studio content, then leave it enabled.
 # .. toggle_use_cases: open_edx
 # .. toggle_creation_date: 2020-11-02
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/25153
```

Comment:
```suggestion
# .. toggle_tickets: https://github.com/edx/edx-organizations/blob/master/docs/decisions/0001-phase-in-db-backed-organizations-to-all.rst
```
Not sure exactly what field is supposed to link to, but maybe the ADR would be more helpful?